### ISSUE TITLE
[Question/Suggestion] Beach Selection Per Climate

### ISSUE THREAD

**Issue by Desymoo** [BlackJar72/ClimaticBiomePlacement#57]
Hello! o/ I know you aren't currently working on this mod so I'll leave this here more as a question than a suggestion 👍 

TLDR: Is there any way to configure what beaches will spawn per climate zone? If not, I'd love to throw it out there as a fun idea for the future in case you ever feel inspired to pick this mod up again :)

---

The long-winded explanation:
After playing around with this mod for a few days now I've honestly been blown away by how cool it is! In particular I really love that you added climate zones to the rivers and oceans, this is the only mod I've found that gives me control over ocean biomes in 1.12 and it's opened up so many cool possibilities for the pack I'm making :D It works beautifully alongside AW2's custom island and ship spawns, really brings the oceans to life.

The one thing that even remotely occurred to me as 'missing' after all my play testing was just the ability to set different beaches to spawn in different climates (if that's possible and I missed it please let me know). This would allow for a bit more compatibility with a few other mods, and as far as I'm aware no other mod really let's you do that for 1.12 (except maybe RTG?). 

Either way, I figured it wouldn't hurt to throw the idea out there :) I'm curious to know if you think this would even be a feasible addition? I know beach generation has historically been difficult to deal with in modded MC.

**Comment by BlackJar72** [comment]
It could be done, I'd have to add it -- for climate zones it would just be adding another split for beaches and temp specific config files.  I'd just need to get around to taking some time to do it, since it would not be hard.

Now, beaches for specific biomes could be hard to do with my system, probably will never do that.

**Comment by BlackJar72** [comment]
I'm not really working on Minecraft mods anymore, but this was an easy addition that I'd wanted to add anyway at one point so I did it.

Note that it does not add modded beaches out of the box -- going through all the supported mods to find all the beaches and decide where they should go actually would be a big jobs and something I'm not up for right now.

**Comment by Desymoo** [comment]
Holy crap, thank you! I certainly wasn't expecting such a quick fix :D

In regards to setting all the modded biomes - I feel it's not too difficult for users to figure out their favorite modded beaches and set those manually on their end. I'm just ecstatic that the option now exists! 

Thank you so much for taking the time to add this, especially for a mod you weren't planning on further supporting. You're awesome :)
